hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/toolkit/ReturnT.java

```diff
@@ -43,13 +43,13 @@
 
     private int code;
 
-    private String msg;
+    private String message;
```

Comment:
Will the transformation here have any impact on other processes?

Reply:
I noticed that in the front-end interceptor, when parsing the back-end data packet, the name of the checked field is "message", so change "msg" to "message".

If you use the old name "msg", without changing the front-end code, the front-end will not be able to obtain the information that the back-end wants to submit to the front-end.